frontend/src/views/DashboardSidebar.vue

```diff
@@ -12,6 +12,13 @@
       <heroicons-outline:shield-exclamation class="w-5 h-5 mr-2" />
       {{ $t("anomaly-center") }}
     </router-link>
+    <router-link
```

Comment:
Put it above anomaly center ? since sql editor is more commonly used

Reply:
Done. PTAL